partcad/src/partcad/logging.py

```
@@ -118,12 +118,14 @@ class Ops:
 
 # Classes to be used with "with()" to alter the logging context.
 class Process(object):
-    def __init__(self, op: str, package: str, item: str = None):
+    def __init__(self, op: str, package: str, item: str = None, transaction=None):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)